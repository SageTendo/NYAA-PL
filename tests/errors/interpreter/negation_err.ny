uWu_nyaa() => {
    yomu(-"1")
}