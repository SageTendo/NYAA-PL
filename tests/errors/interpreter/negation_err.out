bad operand type for unary -